src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/ComponentLayout.razor

```diff
@@ -3,11 +3,12 @@
 
 <div>
     <div class="navbar">
-        <BitLink Href="#">Home</BitLink>
-        <BitLink Href="/component/button" Class="active">Component</BitLink>
-        <BitLink Href="#">Description</BitLink>
-
         <h3 class="header-title">BIT Component</h3>
+        <div class="links">
+            <BitLink Href="#">Home</BitLink>
+            <BitLink Href="/component/button" Class="active">Component</BitLink>
```

Comment:
change all the routes and labels related to component into the plural form: "components"

Reply:
This has been done in another branch and was resolved after merging with develop branch